packages/react/src/Pillar/Pillar.tsx

```diff
@@ -105,12 +116,23 @@ const PillarDescription = forwardRef<HTMLParagraphElement, PillarDescriptionProp
   }
 )
 
+type PillarLinkProps = Omit<LinkProps, 'size' | 'direction'> & BaseProps<HTMLAnchorElement>
```

Comment:
`href` is called out as required in the docs, so could you make it required here too please? 

```suggestion
type PillarLinkProps = {
  href: string
} & Omit<LinkProps, 'size' | 'direction'> &
  BaseProps<HTMLAnchorElement>
```